tsconfig.json

```diff
@@ -1,22 +1,18 @@
 {
-  "compilerOptions": {
-    "target": "es2019",
-    "esModuleInterop": true,
-    "types": ["node", "vitest/globals"],
-    "lib": ["es2020"],
-    "moduleResolution": "node",
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "sourceMap": true,
-    "strict": true,
-    "noImplicitReturns": true,
-    "skipLibCheck": true,
-    "declaration": true,
-    "declarationMap": true,
-    "emitDeclarationOnly": true,
-    "noUncheckedIndexedAccess": true
-  },
-  "ts-node": {
-    "require": ["tsconfig-paths/register"]
-  }
+  "extends": "./tsconfig.options.json",
+  "files": [],
+  "references": [
+    { "path": "./backend/orchestrator/tsconfig.json" },
+    { "path": "./contracts/orchestrator-contracts/tsconfig.build.json" },
+    { "path": "./packages/create-swarmion-app/tsconfig.json" },
+    { "path": "./packages/eslint-plugin/tsconfig.build.json" },
+    { "path": "./packages/nx-plugin/tsconfig.build.json" },
+    { "path": "./packages/serverless-cdk-plugin/tsconfig.build.json" },
+    { "path": "./packages/serverless-configuration/tsconfig.build.json" },
+    { "path": "./packages/serverless-contracts/tsconfig.build.json" },
+    { "path": "./packages/serverless-contracts-plugin/tsconfig.build.json" },
+    { "path": "./packages/serverless-helpers/tsconfig.build.json" },
+    { "path": "./user-docs/cloudfront/tsconfig.json" },
+    { "path": "./user-docs/documentation/tsconfig.json" }
```

Comment:
This means we have to update this file when generating a new package/service right?

Reply:
Yes, that way we can run [`tsc --build --verbose` at the root-level](https://moonrepo.dev/docs/guides/javascript/typescript-project-refs#on-all-projects)

Reply:
This needs to be populated in generators right?

Reply:
I upgrade them!